packages/itwin/ec3-widget/package.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "@itwin/ec3-widget-react",
+  "version": "0.0.0",
+  "description": "An iTwin.js 3D Viewer Widget that interfaces with the EC3 Carbon Calculation.",
+  "keywords": [
+    "Bentley",
+    "BIM",
+    "iModel",
+    "iTwin.js",
+    "Carbon Calculation"
+  ],
+  "main": "lib/cjs/ec3-widget-react.js",
+  "directories": {
+    "test": "jest"
+  },
+  "license": "MIT",
+  "author": {
+    "name": "Bentley Systems, Inc.",
+    "url": "http://www.bentley.com"
+  },
+  "module": "lib/esm/ec3-widget-react.js",
+  "typings": "lib/cjs/ec3-widget-react",
+  "scripts": {
+    "build": "npm run -s dual-build && npm run -s copy:assets",
+    "build:cjs": "tsc 1>&2 --outDir lib/cjs",
+    "build:esm": "tsc 1>&2 --module ES2020 --outDir lib/esm",
+    "clean": "rimraf lib",
+    "copy:assets": "npm run -s copy:public && npm run -s copy:cjs && npm run -s copy:esm",
+    "copy:cjs": "cpx \"./src/**/*.scss\" ./lib/cjs",
+    "copy:esm": "cpx \"./src/**/*.scss\" ./lib/esm",
+    "copy:public": "cpx \"./public/**/*\" ./lib/public",
+    "cover": "npm test -- --coverage",
+    "dual-build": "npm run -s build:cjs && npm run -s build:esm",
+    "lint": "eslint -f visualstudio \"./src/**/*.{ts,tsx}\" 1>&2",
+    "lint:fix": "npm run lint -- --fix",
+    "pseudolocalize": "betools pseudolocalize --englishDir ./public/locales/en --out ./public/locales/en-PSEUDO",
+    "rebuild": "npm run clean && npm run build",
+    "test": "jest",
+    "test:watch": "jest --watch "
+  },
+  "eslintConfig": {
+    "plugins": [
+      "@itwin"
+    ],
+    "extends": "plugin:@itwin/ui",
+    "rules": {
+      "no-duplicate-imports": "off",
+      "@typescript-eslint/consistent-type-imports": "error",
+      "deprecation/deprecation": "off"
+    }
+  },
+  "dependencies": {
+    "@itwin/insights-client": "^0.2.0",
+    "@itwin/itwinui-icons-react": "^1.7.0",
+    "@itwin/itwinui-react": "~1.42.0",
```

Comment:
Change `@itwin/itwinui-react` version to use a caret `^` as per these guidelines: https://github.com/iTwin/iTwinUI-react/wiki/Version-conflicts